### PR TITLE
add Akash Khambay

### PR DESCRIPTION
**Pull request by akashkhambay**
Just added my name.

### CODE REVIEW

File: morris/roster.json [hunk 1]
```diff
@@ -39,5 +39,8 @@
 		{
 			"name": "Maximilian Hartley"
 		}
+		{
```
Comment (Gingertonic) [inline morris/roster.json:42]: We'll need to make sure there is a comma between each object! Please could you add one between yours and Max's?

**Review by Gingertonic — CHANGES_REQUESTED** [review]
Just a missing comma! No need to make a new PR, just push the additional change

File: morris/roster.json (hunk 1 above)
Comment (akashkhambay) [reply morris/roster.json:42]: and ive pushed and committed it

**Review by Gingertonic — APPROVED** [review]
Thanks @akashkhambay :)

**Pull request merged**